apps/backend/src/auth/ProposalAuthorization.ts

```diff
@@ -206,37 +225,44 @@ export class ProposalAuthorization {
       return false;
     }
 
-    const technicalReviews = await this.reviewDataSource.getTechnicalReviews(
-      proposal?.primaryKey
-    );
-
-    const isInternalReviewerOnSomeTechnicalReview = technicalReviews
-      ? (
-          await Promise.all(
-            technicalReviews.map(
-              async (technicalReview) =>
-                await this.userAuth.isInternalReviewerOnTechnicalReview(
-                  agent,
-                  technicalReview.id
-                )
-            )
-          )
-        ).some((value) => value)
-      : false;
+    const currentRole = agent?.currentRole?.shortCode;
+
+    let haveAccess = false;
+
+    switch (currentRole) {
+      case Roles.USER:
+        haveAccess =
+          (await this.isMemberOfProposal(agent, proposal)) ||
+          (await this.isVisitorOfProposal(agent, proposal.primaryKey));
+        break;
+      // case Roles.INSTRUMENT_SCIENTIST:
```

Comment:
These were in the old check but were irrelevant due to the `isInstrumentScientist` check. I think as stfc would be easiest for us to keep the `isInstrumentScientist`. What about anyone else?

Reply:
What you found is a bug. Could you remove the this.userAuth.isInstrumentScientist check and uncomment this part of the code? We want to make sure instrument scientists have the right permissions.